scripts/pr_review_test.py

```
@@ -778,15 +824,296 @@ def test_paths_dropped_by_the_token_budget_still_count_as_fetches(self) -> None:
+                self.returncode = None
+                self.killed = False
+                self.waited = False
+
+            def poll(self) -> int | None:
+                return None
+
+            def kill(self) -> None:
+                self.killed = True
+                self.returncode = -9
+
+            def wait(self, timeout: float | None = None) -> int:
```

Ruff (0.16.1)

[warning] Unused method argument: `timeout`

(ARG002)